spec/rails_helper.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'simplecov'
+
+SimpleCov.start do
+  add_filter 'config/'
+end
+
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../config/environment', __dir__)
+
+abort('The Rails environment is running in production mode!') if Rails.env.production?
+require 'rspec/rails'
+
+Dir[Rails.root.join('spec/**/support/**/*.rb')].sort.each { |f| require f }
+Dir[Rails.root.join('spec/**/concerns/**/*.rb')].sort.each { |f| require f }
+Dir[Rails.root.join('spec/**/services/**/*.rb')].sort.each { |f| require f }
+
+begin
+  ActiveRecord::Migration.maintain_test_schema!
+rescue ActiveRecord::PendingMigrationError => e
+  puts e.to_s.strip
+  exit 1
+end
+RSpec.configure do |config|
+  config.include FactoryBot::Syntax::Methods
+  config.include Devise::Test::ControllerHelpers, type: :controller
+  config.include ControllerHelpers, type: :controller
+  config.include FeatureHelpers, type: :feature
+  config.include MailerHelpers, type: :mailer
```

Comment:
```suggestion
  config.include TranslateHelpers
```

You defined the same method 3 times because you wanted to separate injecting of the method to particular spec types (only controller, feature, mailer, nothing else). That's fine if the method could be present in other spec types and could lead to conflicts. However, this particular method you introduced is commonly used for translations. So I guess you can at least consider the possibility to inject this for all specs altogether.